tests/lib/Server/Output/ValueObjectVisitor/RestExecutedViewTest.php

```diff
@@ -115,7 +117,7 @@ protected function internalGetVisitor()
     {
         return new ValueObjectVisitor\RestExecutedView(
             $this->getLocationServiceMock(),
-            $this->getContentServiceMock(),
+            new RelationListHelper($this->getContentServiceMock())
```

Comment:
So... rather mock it here and unit-test in ibexa/core, as reviewed in https://github.com/ibexa/core/pull/444#pullrequestreview-2405783235